e107_plugins/forum/templates/forum_post_template.php

```diff
@@ -13,21 +13,35 @@
 // New in v2.x - requires a bootstrap theme be loaded.  
 
 //$FORUM_POST_TEMPLATE['caption']		= "Custom caption";
+
+$SC_WRAPPER['FORUM_POST_AUTHOR'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_3010."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+$SC_WRAPPER['FORUM_POST_SUBJECT'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_3011."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+
+// Replaces legacy? {FORUM_POST_OPTIONS_LABEL} shortcode...
+$SC_WRAPPER['FORUM_POST_OPTIONS'] = "<div class='form-group'>
+										<label for='name' class='col-sm-3 control-label'>".LAN_FORUM_8013."</label>
+										 <div class='col-sm-9'>{---}</div>
+									</div>
+";
+
+	
```

Comment:
I know $SC_WRAPPER is deprecated, or better, for legacy themes, but i try to use the $FORUM_POST_WRAPPER and it doesn't works....
So i stick to the legacy one....